app/main/routes.py

```diff
@@ -113,6 +124,58 @@ def add_raid_player():
         flash('RaidPlayer added successfully!')
         return redirect(url_for('main.debug'))
 
+@bp.route('/signups/populate')
+def populate():
+    server_id = "955055697144446976"
+    api_key = os.getenv('RAID_HELPER_API_KEY')
+
+    events = fetch_all_raid_events(server_id, api_key)
+
+    if events is None:
+        return "Failed to fetch events from the API.", 500
+
+    posted_events = events.get('postedEvents', [])
+    
+    if not posted_events:
+        return "No events found.", 404
+
+    latest_event = posted_events[0]
+    event_id = latest_event.get('id')
+
+    detailed_event = fetch_event_details(event_id)
+
+    if detailed_event is None:
+        return f"Failed to fetch details for event ID: {event_id}", 500
+
+    signups = detailed_event.get('signUps', [])
+
+    # Clear the SignUp table before populating
```

Comment:
Hmm, I'm guessing this is for debugging purposes? Later we probably want to have something more in line with: if !exist then create else update entry.